src/BreadcrumbItem.js

```diff
@@ -49,22 +54,24 @@ const BreadcrumbItem = React.forwardRef(
       // Need to define the default "as" during prop destructuring to be compatible with styled-components github.com/react-bootstrap/react-bootstrap/issues/3595
       as: Component = 'li',
       linkAs: LinkComponent = SafeAnchor,
+      linkProps,
       ...props
     },
     ref,
   ) => {
     const prefix = useBootstrapPrefix(bsPrefix, 'breadcrumb-item');
 
     const { href, title, target, ...elementProps } = props;
-    const linkProps = { href, title, target };
+    const linkComponentProps = { href, title, target, ...linkProps };
 
     return (
       <Component
         ref={ref}
         className={classNames(prefix, className, { active })}
         aria-current={active ? 'page' : undefined}
+        {...elementProps}
       >
-        {active ? children : <LinkComponent {...elementProps} {...linkProps} />}
+        {active ? children : <LinkComponent {...linkComponentProps} />}
```

Comment:
```suggestion
        {active ? (
          children
        ) : (
          <LinkComponent
            {...linkProps}
            href={href}
            title={title}
            target={target}
          />
            {children}
          </LinkComponent>
        )}
```

Reply:
am i mis-reading or did we somehow drop `children` as an input to `linkProps`?

Reply:
yeah, woof:
![image](https://user-images.githubusercontent.com/3112159/77174699-d850c780-6a97-11ea-9290-04c20435ed8c.png)